subxt/src/client/light_client/rpc.rs

```diff
@@ -28,7 +44,16 @@ impl LightClientRpc {
     ///
     /// ## Panics
     ///
-    /// Panics if being called outside of `tokio` runtime context.
+    /// The panic behaviour depends on the feature flag being used:
+    ///
+    /// ### Native
+    ///
+    /// Panics when called outside of a `tokio` runtime context.
+    ///
```

Comment:
Same comment as others :)

Reply:
Think this should be updated now 🤔 Might be a github race somewhere